unison-src/transcripts/delete.md

```diff
@@ -98,3 +98,112 @@ structural type foo = Foo ()
 ```ucm
 .> delete.verbose foo
 ```
+
+We want to be able to delete multiple terms at once
+
+```unison:hide
+a = "a"
+b = "b"
+c = "c"
+```
+
+```ucm
+.> add
+.> delete.verbose a b c
+```
+
+We can delete terms and types in the same invocation of delete
+
+```unison:hide
+structural type Foo = Foo ()
+a = "a"
+b = "b"
+c = "c"
+```
+
+```ucm
+.> add
+.> delete.verbose a b c Foo
+.> delete.verbose Foo.Foo
+```
+
+We can delete a type and its constructors
+
+```unison:hide
+structural type Foo = Foo ()
+```
+
+```ucm
+.> add
+.> delete.verbose Foo Foo.Foo
+```
+
+You should not be able to delete terms which are referenced by other terms
+
+```unison:hide
+a = 1
+b = 2
+c = 3
+d = a + b + c
+```
+
+```ucm:error
+.> add
+.> delete.verbose a b c
+```
+
+But you should be able to delete all terms which reference each other in a single command
+
+```unison:hide
+e = 11
+f = 12 + e
+g = 13 + f
+h = e + f + g
+```
+
+```ucm
+.> add
+.> delete.verbose e f g h
+```
+
+You should be able to delete a type and all the functions that reference it in a single command
+
+```unison:hide
+structural type Foo = Foo Nat
+
+incrementFoo : Foo -> Nat
+incrementFoo = cases
+  (Foo n) -> n + 1
+```
+
+```ucm
+.> add
+.> delete.verbose Foo Foo.Foo incrementFoo
+```
+
+If you mess up on one of the names of your command, delete short circuits
+
+```unison:hide
+e = 11
+f = 12 + e
+g = 13 + f
+h = e + f + g
+```
+
+```ucm:error
+.> add
+.> delete.verbose e f gg
+```
+
+Cyclical terms which are guarded by a lambda are allowed to be deleted
```

Comment:
Is this still behaviour we want now that we could potentially require that they all be deleted at once?

I think before, having `delete` only take a single argument meant it was _impossible_ to delete them otherwise, but now that restriction is lifted.